Header header

# "True" heading in degrees
float32 true_heading
# Magnetic heading in degrees
float32 mag_heading

# Water speed in knots
float32 knots
# Water speed in km/h
float32 kmph
# Water speed in m/s
float32 mps